etc/helm/examples/enterprise-values.yaml

```diff
@@ -1,7 +1,24 @@
+# SPDX-FileCopyrightText: Pachyderm, Inc. <info@pachyderm.com>
+# SPDX-License-Identifier: Apache-2.0
 deployTarget: LOCAL
+
 console:
   enabled: false
 enterpriseServer:
   enabled: true
 pachd:
   enabled: false
+  activateEnterprise: true
+  enterpriseLicenseKey: "<ENTEPRRISE-LICENSE-KEY>"
+  oidc:
+    clientId: "enterprise-pach"
+    issuer: "http://<ENTERPRISE-SERVER-IP>:31658/"
+    redirectURI: "http://<ENTERPRISE-SERVER-IP>:31657/authorization-code/callback"
+    setupConnector: true
+    mockIDP: false
+    connector:
+      id: "idp"
```

Comment:
Is there ever a case where we'd have multiple connectors? (ie multiple social oauth providers?)

Reply:
excellent question. based on the data structure it appears possible but I'm not sure what that would look like. Let me try that locally